package.json

```diff
@@ -25,6 +25,7 @@
         "acceptancetest:erc20": "ts-mocha packages/server/tests/acceptance/index.spec.ts  -g '@erc20' --exit",
         "acceptancetest:htsprecompile": "ts-mocha packages/server/tests/acceptance/index.spec.ts  -g '@htsprecompile' --exit",
         "acceptancetest:htsprecompilev1": "ts-mocha packages/server/tests/acceptance/index.spec.ts  -g '@htsprecompilev1' --exit",
+        "acceptancetest:htsprecompilev2": "ts-mocha packages/server/tests/acceptance/index.spec.ts  -g '@htsprecompilev2' --exit",
```

Comment:
the -g flag is a grep and thus `-g '@htsprecompile'` will match three acceptance tests (acceptancetest:htsprecompile, acceptancetest:htsprecompilev1, acceptancetest:htsprecompilev2).